docs/book/common_code/images/activity_machineset_reconciliation.drawio

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-25T21:52:19.710Z" host="www.draw.io" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" etag="lrg_vSUQDvgefqgiXO4p" version="10.5.5" type="google"><diagram id="7pWM9kC39Dyr3UhoOzVi" name="Page-1">7Vxbd5s4EP41fkwPSNz82FyantNmu5s8bPOIQRgajFyQa3t//UogDEi+yDbG2EkeWjRIgEffzHwzjD2Ad5PFY+pOwyfso3gANH8xgPcDAHR96ND/mGTJJZpjFJJxGvlcVgleov9QOZFLZ5GPssZEgnFMomlT6OEkQR5pyNw0xfPmtADHzbtO3TGSBC+eG8vSfyOfhIXUMbVK/hVF47C8s67xMxO3nMwFWej6eF4TwYcBvEsxJsXRZHGHYqa9Ui/Fui8bzq4eLEUJUVlg/lq8jn6Gnh/85d7+kz49v35f3PCr/HHjGf/A/GHJstQAvQpVNh3chmQSU5lOD+mHmbLzGXFT8kJcws4HURzf4Rin+UKo5X9sMknxG6qdCQJ+Rv4M5QOhlKBFTcQ/0yPCE0TSJZ1SntXMYglHmMWH82q3IN+BsLZRpczl+BivLlypkB5wLe6hUaCgUZ9CjA9xSkI8xokbP1TSuqaZJiKKx89xNE6obIQJwRN6AiX+ZwZwdpEpSgoJNyBnq9apstPlTyrUPpnl8JUNy8H9ojFa8tHG3crwLPXQbphRsIwR2TIPFvOYgrbufYpil0R/mja6biv50r9xRB95hRlHa0BGtwQwFM/JF1V4oOp2l7VpUzYh23wbHQj3AU7TQvebTw+KJ6jAuVLJ4XiFEl4fEcn9lxdGCcryAR79Ys5VBHKKZ4mPfI7UeRgR9DJ1cyTMaTxoAtlNPY5Og4EpwAlZ5ywEJxLQP+/kTsQSFG+AT7IbWW1P3Y8Yp/Ijxocf2QjWnX7E7MaPiAYLKtzscCVtWa8poSR2RzkHy1BMTZbuWU5FvJCRqC8DYMVUb7cjKrbGZLUXdZsO8WQ0y3bb81kM1RYUrtvrDFVbY6j2qQzVkrYgwceZqstNNEYBObHl6g27rcy4dcs1FS23dHM7TZcDoqQWR1qyJVICTT8NJ7A0AcEm3MoJxPm6BjsgBbaE6SXKTgpqgqcXFotUEe0cBuhlk18cHamal4XAVoxU0qVumvnWDRAMBQdBThrbD3aOhMpxzk69eJYRlObpfSpQ13fCWCE4N18dSptznL+4Cr7qKPqIMth0TVgN0C1dLT9nDSWV8aJFlLFnpdTUnTBTLP7dNExGWTHW8DxB6TMK6ms5uWXR87oor1Hq8GyEV5eLhhfEeLVBR4y33Kfdxq+arbZLeVehoASW0bEvkCulHyTzYBDBYT9oZhNTplhm7z/J1OWSCq+BrokyGyKUVqwYxdh7+8GW3aMYkQgn18tHJW9ydkKqy4WZD0a6gnd7DuVIlwGFAoepSEn3LbxI99E0ngGrvo45YAUAzRWnqdbocmJMfUBBqnkuTI9uv/+4+/Z+HJBpOud2QFecEtNcjdSW0dErf0h2XC1igxaclmoeDTrKo6FQfjEteGg5TeFShX7a94BDsMOf7VxxGn8G5CpBtkw8KnlG05hawPvzZpaxJuvv1JsBOeu/Gm92uGcCqvkZ6Ki1RTRZ29A6fiVd4uLgnjK22+fsKDPMZvKpa2BdO0i3XWVym85s6jMlCZ06GdXcTK6iXK1jtIF29lYdcFG9Ol3VP4Fqsw6wunGNhpCdWUNT0TVuZlqtO0+5aJHNPA9R07309xii+m3NPHvvDpAbHS7oVcaezTtlzlZmabVVJ8jZSiT3tePHdoS+cUfVG+zkXI4uca6WUjixG8g29K1Pdp7uISDXo671zU63dRB1m+rF2yBLKG3Ypmou0qP3QUCu4gWYbRNTDy9LuEFRbH2Kkmfk+ssX5OHELxpuLytGS54MyhHaWBOhaY50ohAN5WLQR4huyZ0M+x2iHdhEo2FAJe+xb1Q1hZzSsbaXRcWouu/8bqIwlAt21xqFOzABaPcxojq2cIn+x1Mop1wp+j1DdHC1RatVzx5X99A8//fLoEzSj/MNV1HPL9HZXkhsOwZCtRjYmrnK9JfRLy2Okrf1Ralme27V3ruzaSq/rs86pRi5JtEEZYSdvCwKbTU9qwnWVrn0tRza3IyO475LetEcurOKNVQlA/qB3+k5tmNXbC+yun63Z3xwSnUYCehQwJUBzsApxaY1C6p+iblHrNIAEjBd36/HEx9R3Pgof9wgonCk4GFhKWINvQlCjGheLQEVC3E9eGlqyG+0e8w/uwpBJY57wz9F72BD1Zhz9K+laNA+rtuMDqufVyqmV79SBR/+Bw==</diagram></mxfile>
```

Comment:
This diagram seems to reference two blocks that I don't see diagrams for:
- filter machine BLOCK
- sync Replica BLOCK

Reply:
I was hoping that we can just add description for those.  I don't think these diagrams need to document every single code, but if everyone thinks these are important, we can add diagrams for them.

Reply:
I think that is fine as well, I just wanted to make sure there was a reason for the exclusion. Can we add a statement to the book content to say that they are referenced but not currently diagrammed?

Reply:
@detiber added